#We Got Coders - My Amazing Application.html#

```diff
@@ -0,0 +1,176 @@
+<!DOCTYPE html>
```

Comment:
Meanwhile, I've find out, which files is relevant. So, I'm gonna delete this one.